### PR TITLE
Refresh the network aruba instant snmp documentation

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md [hunk 1]
```diff
@@ -12,152 +12,376 @@ Aruba Networks est un fournisseur de solutions d'accès réseau.
 
 Le connecteur de supervision *Aruba Instant SNMP* utilise le protocole SNMP pour se connecter, récupérer des informations et les métriques relatives aux bornes d'accès sans fil de marque Aruba.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md:13]:
```suggestion
Le connecteur de supervision *Aruba Instant SNMP* utilise le protocole SNMP pour se connecter et récupérer des informations et des métriques relatives aux bornes d'accès sans fil de marque Aruba.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md [hunk 2]
```diff
@@ -12,152 +12,376 @@ Aruba Networks est un fournisseur de solutions d'accès réseau.
...
+2. Renseignez les macros désirées (par exemple, ajustez les seuils d'alerte). Les macros indiquées ci-dessous comme requises (**Obligatoire**) doivent être renseignées.
+
+<Tabs groupId="sync">
+<TabItem value="Ap-Usage" label="Ap-Usage">
+
+| Macro                | Description                                                                                                                                              | Valeur par défaut  | Obligatoire |
+|:---------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------|:-----------:|
+| FILTERNAME           | Filter access point name (can be a regexp)                                                                                                               |                    |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md:195]: unclear

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md [hunk 3]
```diff
@@ -12,152 +12,376 @@ Aruba Networks est un fournisseur de solutions d'accès réseau.
...
+| WARNINGSTATUS        | Define the conditions to match for the status to be WARNING (default: ''). You can use the following variables: %{status}, %{display}                    |                    |             |
+| EXTRAOPTIONS         | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                                                       | --verbose          |             |
+
+</TabItem>
+<TabItem value="Ssid-Status" label="Ssid-Status">
+
+| Macro          | Description                                                                                                                                                  | Valeur par défaut      | Obligatoire |
+|:---------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------|:-----------:|
+| FILTERNAME     | Filter SSID name (can be a regexp)                                                                                                                           |                        |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md:215]: unclear

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md [hunk 4]
```diff
@@ -12,152 +12,376 @@ Aruba Networks est un fournisseur de solutions d'accès réseau.
...
-	--snmp-community='test/aruba' \
+	--snmp-community='my-snmp-community'  \
 	--filter-name='' \
 	--warning-status='' \
 	--critical-status='%{status} !~ /up/i' \
 	--warning-cpu='' \
 	--critical-cpu='' \
-	--warning-clients='20' \
-	--critical-clients='50' \
+	--warning-clients='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md:245]: why did the the thresholds disappear?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md [hunk 5]
```diff
@@ -12,152 +12,376 @@ Aruba Networks est un fournisseur de solutions d'accès réseau.
...
 
-### UNKNOWN: SNMP GET Request : Timeout
+<Tabs groupId="sync">
+<TabItem value="Ap-Usage" label="Ap-Usage">
 
-Si vous obtenez ce message, cela signifie que le collecteur Centreon ne parvient pas à contacter votre borne d'accès sans fil Aruba sur le port UDP 161, ou alors que la communauté SNMP configurée n'est pas correcte.
+| Option                   | Description                                                                                                                                                |
+|:-------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --filter-counters        | Only display some counters (regexp can be used). Example: --filter-counters='^cpu$'                                                                        |
+| --filter-name            | Filter access point name (can be a regexp).                                                                                                                |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md:362]: unclear

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md [hunk 6]
```diff
@@ -12,152 +12,376 @@ Aruba Networks est un fournisseur de solutions d'accès réseau.
...
+| --warning-* --critical-* | Thresholds. Can be: 'total-ap', 'cpu', 'clients', 'mem-usage' (B), 'mem-usage-free' (B), 'mem-usage-prct' (%).                                             |
 
-### UNKNOWN: SNMP GET Request : Cant get a single value.
+</TabItem>
+<TabItem value="Ssid-Status" label="Ssid-Status">
 
-Si vous rencontrez cette erreur, il est probable que les autorisations données à l'agent SNMP soient trop restreintes. Ce dernier doit avoir accès à la branche entreprise Aruba: .1.3.6.1.4.1.14823
+| Option            | Description                                                                                                                                                     |
+|:------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --filter-name     | Filter SSID name (can be a regexp).                                                                                                                             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md:372]: unclear

File: pp/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md [hunk 7]
```diff
@@ -10,39 +10,65 @@ import TabItem from '@theme/TabItem';
 
 Aruba Networks is a provider of network access solutions for enterprises.
 
-The Centreon Monitoring Connector *Aruba Instant SNMP* relies on the SNMP protocol to query and collect status and metrics of the Aruba Wireless Access Point equipments. 
+The Centreon Monitoring Connector *Aruba Instant SNMP* relies on the SNMP protocol to query and collect status and metrics of the Aruba Wireless Access Point equipments.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md:13]:
```suggestion
The Centreon Monitoring Connector *Aruba Instant SNMP* relies on the SNMP protocol to query and collect status and metrics for the Aruba Wireless Access Point devices.
```

File: pp/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md [hunk 8]
```diff
@@ -57,115 +83,309 @@ The SNMP agent must be configured and running on the Aruba device. Please refer
...
+
+1. If you have used a host template and checked **Create Services linked to the Template too**, the services linked to the template have been created automatically, using the corresponding service templates. Otherwise, [create manually the services you want](/docs/monitoring/basic-objects/services) and apply a service template to them.
+2. Fill in the macros you want (e.g. to change the thresholds for the alerts). Some macros are mandatory (see the table below).
+
+<Tabs groupId="sync">
+<TabItem value="Ap-Usage" label="Ap-Usage">
+
+| Macro                | Description                                                                                                                                              | Default value      | Mandatory   |
+|:---------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------|:-----------:|
+| FILTERNAME           | Filter access point name (can be a regexp)                                                                                                               |                    |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md:201]: unclear

File: pp/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md [hunk 9]
```diff
@@ -57,115 +83,309 @@ The SNMP agent must be configured and running on the Aruba device. Please refer
...
 
-### UNKNOWN: SNMP GET Request : Timeout
+<Tabs groupId="sync">
+<TabItem value="Ap-Usage" label="Ap-Usage">
 
-If you get this error, it may indicate that some flows are blocked between the Centreon Poller and the Aruba devices. 
+| Option                   | Description                                                                                                                                                |
+|:-------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --filter-counters        | Only display some counters (regexp can be used). Example: --filter-counters='^cpu$'                                                                        |
+| --filter-name            | Filter access point name (can be a regexp).                                                                                                                |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md:366]: unclear

File: pp/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md [hunk 10]
```diff
@@ -57,115 +83,309 @@ The SNMP agent must be configured and running on the Aruba device. Please refer
...
+| WARNINGSTATUS        | Define the conditions to match for the status to be WARNING (default: ''). You can use the following variables: %{status}, %{display}                    |                    |             |
+| EXTRAOPTIONS         | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).                                                       | --verbose          |             |
+
+</TabItem>
+<TabItem value="Ssid-Status" label="Ssid-Status">
+
+| Macro          | Description                                                                                                                                                  | Default value          | Mandatory   |
+|:---------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------|:-----------:|
+| FILTERNAME     | Filter SSID name (can be a regexp)                                                                                                                           |                        |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md:221]: unclear

File: pp/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md [hunk 11]
```diff
@@ -57,115 +83,309 @@ The SNMP agent must be configured and running on the Aruba device. Please refer
...
+| --warning-* --critical-* | Thresholds. Can be: 'total-ap', 'cpu', 'clients', 'mem-usage' (B), 'mem-usage-free' (B), 'mem-usage-prct' (%).                                             |
 
-It can also mean that the Centreon Host Configuration doesn't reflect the SNMP configuration on Aruba side (version, community). 
+</TabItem>
+<TabItem value="Ssid-Status" label="Ssid-Status">
 
-### UNKNOWN: SNMP GET Request : Cant get a single value.
+| Option            | Description                                                                                                                                                     |
+|:------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --filter-name     | Filter SSID name (can be a regexp).                                                                                                                             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md:376]: unclear

File: pp/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md [hunk 12]
```diff
@@ -57,115 +83,309 @@ The SNMP agent must be configured and running on the Aruba device. Please refer
...
-	--snmp-community='test/aruba' \
+	--snmp-community='my-snmp-community'  \
 	--filter-name='' \
 	--warning-status='' \
 	--critical-status='%{status} !~ /up/i' \
 	--warning-cpu='' \
 	--critical-cpu='' \
-	--warning-clients='20' \
-	--critical-clients='50' \
+	--warning-clients='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md:249]: thresholds?

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**